Core/Systems/EcsSystems.cs

```diff
@@ -9,13 +9,21 @@ public sealed partial class EcsSystems
 		private readonly EcsWorld _world;
 		private readonly List<IEcsSystem> _allSystems = new();
 
+#if SECS_PROFILING
```

Comment:
I suggest naming it SECS_ENABLE_PROFILING, I think it reflects condition better. Similar to SECS_ENABLE_EVENTS.

Also it makes sense to put inside `#if UNITY_EDITOR` as well since it relative only to that engine

Reply:
no unity editor macro cuz for now profiler can be used for dev and rel builds and using that macro would just